contribute/style-guides/backend.md

```diff
@@ -1,6 +1,6 @@
 # Backend style guide
 
-Grafanas backend has been developed for a long time with a mix of code styles.  This guide explains how we want to write Go code in the future.
+Grafana's backend has been developed for a long time with a mix of code styles.  This guide explains how we want to write Go code in the future.
```

Comment:
```suggestion
Grafana's backend has been developed for a long time with a mix of code styles. This guide explains how we want to write Go code in the future.
```

Reply:
Thanks @oddlittlebird, I also noticed the two spaces, but wasn't sure if we have standardized on a single space after a period. Can we add to the style guide that there should be a single space after a period? I know people come from different traditions on this, so it would be great to standardize.